shell_sort/ShellSort.java

```diff
@@ -0,0 +1,41 @@
+public class ShellSort {
+	/**
+	* sorting function
+	* Worst case time complexity = O(n^2)
+	* Best case complexity = O(nlog(n))
+	* n is input size
+	*/
```

Comment:
There's a `Tab`.
try selecting this space inside the double quotes
TAB => "	"
SPACE => "    "